src/rules/no_var.rs

```diff
@@ -63,5 +81,32 @@ mod tests {
       r#"var someVar = "someString"; const c = "c"; let a = "a";"#,
       0,
     );
+
+    assert_lint_err!(
+      NoVar,
+      "var foo = 'bar';": [{
+        col: 0,
+        message: MESSAGE,
+      }],
```

Comment:
Would be great to let this `assert_lint_err!` macro contain all invalid test cases.

```suggestion
    assert_lint_err!(
      NoVar,
      r#"var someVar = "someString"; const c = "c"; let a = "a";"#: [{
        col: 0,
        message: MESSAGE,
      }],
      "var foo = 'bar';": [{
        col: 0,
        message: MESSAGE,
      }],
```

Reply:
Thank you, I think it should cover all of these invalid cases. But I think it already covers in the following lines though, or I am missing something

Reply:
Ah yes, my point has been addressed in your fix 459fa16. I just meant to remove `assert_lint_err::<NoVar>(...)` so it looks totally fine now :)